src/pages/Availabilities.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import DoctorAvailabilities from "../components/DoctorAvailabilities";
+import {useDoctorHooks} from '../hooks/DoctorHooks';
+import {Grid} from "@material-ui/core";
+
+
+
+export default  function Availabilities() {
+    const {doctors} = useDoctorHooks()
+   
+  
+    return (
+      <Grid container direction="row" spacing={2} justify="flex-start" alignItems="center">
+        {doctors.map((doctor, index) => doctor && <DoctorAvailabilities doctor={doctor} key={index}/>)}
```

Comment:
Is there a case where doctor is null?
```suggestion
        {doctors.map((doctor) => <DoctorAvailabilities doctor={doctor} key={doctor.id}/>)}
```

Reply:
I did this code expecting that I will have a loading time on doctors, but I should have removed it because doctors is an empty list at the start not an object, I was wrong because I was reasoning on the item instead of the array.